.github/workflows/sync_pytorch_canary.yml

```diff
@@ -0,0 +1,29 @@
+name: sync_pytorch_canary
+
+on:
+  schedule:
+    # Every day at 3:33am ET
+    - cron: "33 7 * * *"
```

Comment:
nit: Should we have it run more frequently?

Reply:
agree, the workflow should be pretty light so even something like every 15 minutes seems like it'd be fine